apollo-router/src/plugins/telemetry/metrics/prometheus.rs

```diff
@@ -131,10 +132,14 @@ impl MetricsConfigurator for Config {
             .with_registry(registry.clone())
             .build()?;
 
-        let meter_provider = MeterProvider::builder()
+        let mut meter_provider_builder = MeterProvider::builder()
```

Comment:
Not on this line, but further up in the file we check to see if the prometheus registry can be reused. If any of the views change then I think that we need to say it can't be reused.